PWGHF/HFC/Tasks/taskCorrelationLcHadrons.cxx

```diff
@@ -148,8 +169,20 @@ struct HfTaskCorrelationLcHadrons {
     registry.get<THnSparse>(HIST("hCorrel2DVsPtMcGen"))->Sumw2();
   }
 
+  Region getRegion(double deltaPhi)
+  {
+    if (std::abs(deltaPhi) < o2::constants::math::PI / 3.) {
+      return Toward;
+    } else if (deltaPhi > 2. * o2::constants::math::PI / 3. && deltaPhi < 4. * o2::constants::math::PI / 3.) {
+      return Away;
+    } else {
+      return Transverse;
+    }
+  }
```

Comment:
This function is already defined in `taskCorrelationD0Hadrons.cxx`. Please move it out into a common header.

Reply:
Same for the definition of `Region`.

Reply:
Thank you for your suggestion; I’ll address it as soon as possible. Also, regarding the getRegion function, should I create a new header file, or should I include it in an existing one?

Reply:
I would suggest a new header `PWGHF/HFC/Utils/utilsCorrelations.h`

Reply:
I have tried to add it in a new header PWGHF/HFC/Utils/utilsCorrelations.h, but I am not sure if it meets your suggestions.

Reply:
Thanks, it looks good.

Reply:
What do you mean?